### PR TITLE
Hindi translation of webgl contribution guide

### PR DESCRIPTION
**Pull request by ranaaditya**
Resolves Hindi Translation of WebGL Contribution Guide

 Changes: **Add webgl_contribution_guide.md in contributor_docs/hi/ folder**

 Screenshots of the change:

#### PR Checklist

- [x] `npm run lint` passes
- [x] [Inline documentation] is included / updated
- [x] [Unit tests] are included / updated

[Inline documentation]: https://github.com/processing/p5.js/blob/main/contributor_docs/inline_documentation.md
[Unit tests]: https://github.com/processing/p5.js/tree/main/contributor_docs#unit-tests

### CODE REVIEW

**Comment by ranaaditya** [comment]
@TanviKumar @Divyansh013 can you please review this PR and let me know the changes if any.

Thanks !

**Review by Divyansh013 — CHANGES_REQUESTED** [review]
Hey @ranaaditya have left some reviews please look into that. Thanks for your PR :)

File: contributor_docs/hi/webgl_contribution_guide.md
Comment (ranaaditya) [reply contributor_docs/hi/webgl_contribution_guide.md:8]: resolved :+1:

File: contributor_docs/hi/webgl_contribution_guide.md
Comment (ranaaditya) [reply contributor_docs/hi/webgl_contribution_guide.md:9]: resolved :+1:

**Comment by ranaaditya** [comment]
Hi @Divyansh013 I have fixed the suggested changes, can you review and let me know if something needs to be fixed in it.

Thanks,
Aditya

**Review by Divyansh013 — APPROVED** [review]

**Comment by Qianqianye** [comment]
Thanks @ranaaditya for translating and @Divyansh013 for reviewing. I will merge it after @TanviKumar approves the translation since all the translations need to be approved by 2 translation stewards. Thanks!

**Review by TanviKumar — APPROVED** [review]

**Comment by TanviKumar** [comment]
@Qianqianye this PR can be merged

**Comment by Qianqianye** [comment]
Looks great, thanks @ranaaditya @Divyansh013 @TanviKumar  💗
@all-contributors please add @ranaaditya for translation

**Pull request merged**
